### PR TITLE
fix: log and reminder changes

### PR DESCRIPTION
**Pull request by Daljit-fleetHawks**
## Change Description

update odometer/completion date in reminder and log changes

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/pages/fleet/geofence/geofence-list/geofence-list.component.ts [hunk 1]
```diff
@@ -207,10 +207,7 @@ export class GeofenceListComponent implements OnInit {
     this.spinner.show();
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/geofence/geofence-list/geofence-list.component.ts]: Can you remove the spinner. We are not showing spinner.

File: src/app/pages/fleet/maintenance/service-log/service-list/service-list.component.ts [hunk 2]
```diff
@@ -140,10 +139,13 @@ export class ServiceListComponent implements OnInit {
     this.spinner.show();
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/maintenance/service-log/service-list/service-list.component.ts]: Please remove spinner

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]

**Review by kunalfleethawks — APPROVED** [review]

**Comment by kunalfleethawks** [comment]
please resolve conflicts

**Comment by Daljit-fleetHawks** [comment]
> please resolve conflicts

Done

**Pull request merged**